### PR TITLE
#7521 - outlook. commands reviewed and updated

### PR DESCRIPTION
**Pull request by Vegebutcher**
outlook.getfolder, outlook.getrootfolders, outlook.reply and outlook.selectitem written from scratch

### CODE REVIEW

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookReplyCommand.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+In this example, the robot gets the content of the Outlook Inbox folder, assigns mails to the `♥mails` variable, then prepares a reply to the first unread email message (starting from the oldest unread element in the Inbox). The resulting draft reply is assigned to the `♥originalMail` variable. Since it is of `outlookmail` structure, you can retrieve the message body (the conversation history, because it’s a reply) and add a reply content to it: here, it’s done by adding the `♥replyBody` to the `♥originalMailBody` variables. Then, a new message is created with the `outlook.newmessage` command, using the already existing sender and recipient addresses prepared by the `outlook.reply` command:
+
+```G1ANT
+♥outlookInboxFolder = john.doe@g1ant.com\Inbox
+outlook.open
+outlook.getfolder ♥outlookInboxFolder result ♥inboxFolder
+♥mails = ♥inboxFolder⟦unread⟧
+outlook.reply ♥mails⟦1⟧ result ♥originalMail
+♥originalMailBody = ♥originalMail⟦body⟧
+♥replyBody = Hi,⊂"\r\n"⊃Thanks for your email. We will contact you shortly.⊂"\r\n"⊃Regards,⊂"\r\n"⊃G1ANT⊂"\r\n"⊃⊂"\r\n"⊃
```
Comment (DanielWilczek) [inline G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookReplyCommand.md]: ♥replyBody = Hi,⊂"\r\n"⊃Thanks for your email. We will contact you shortly.⊂"\r\n"⊃Regards,⊂"\r\n"⊃G1ANT⊂"\r\n\r\n"⊃

Comment (DanielWilczek) [reply G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookReplyCommand.md]: or something like:
♥replyBody = ⊂"Hi,\r\nThanks for your email. We will contact you shortly.\r\nRegards,\r\nG1ANT\r\n\r\n"⊃

Comment (Vegebutcher) [reply G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookReplyCommand.md]:
```suggestion
♥replyBody = ⊂"Hi,\r\nThanks for your email. We will contact you shortly.\r\nRegards,\r\nG1ANT\r\n\r\n"⊃
```

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookOpenCommand.md [hunk 2]
```diff
@@ -1,30 +1,33 @@
...
 
-For more information about `if`, `timeout`, `errorjump` and `errormessage` arguments, please visit [Common Arguments](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Common-Arguments.md)  manual page.
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
 
-This command is contained in **G1ANT.Addon.MSOffice.dll**.
-See: [https://github.com/G1ANT-Robot/G1ANT.Addon.MSOffice](https://github.com/G1ANT-Robot/G1ANT.Addon.MSOffice)
+## Example
 
-**Example 1:**
+This simple script just opens Outlook and then brings its window to the foreground (if it’s not already):
```
[the remark this answers is not in the document]
Comment (Vegebutcher) [reply G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookOpenCommand.md]:
```suggestion
This simple script just opens Outlook and then brings its window to the foreground (if it’s not there already):
```

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookCloseCommand.md [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
 
-For more information about `if`, `timeout`, `errorjump` and `errormessage` arguments, please visit [Common Arguments](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Common-Arguments.md)  manual page.
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
 
-This command is contained in **G1ANT.Addon.MSOffice.dll**.
-See: [https://github.com/G1ANT-Robot/G1ANT.Addon.MSOffice](https://github.com/G1ANT-Robot/G1ANT.Addon.MSOffice)
+## Example
 
-**Example 1:**
+In this simple example Outlook is closed and the result — *true* or *false* — is displayed in a dialog box:
```
[the remark this answers is not in the document]
Comment (Vegebutcher) [reply G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookCloseCommand.md]:
```suggestion
In this simple example Outlook is closed and the result — *true* or *false* as to whether the command succeeded — is displayed in a dialog box:
```

**Pull request merged**